app.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require_relative 'classroom'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
```

Comment:
- Hey, great job with your app's UI implementation, everything works as expected, however, your app fails to run on another machine because the file path is case-sensitive. You have named your files as 
`Person, Student, and Teacher`, but you've accessed them with `person, teacher, and student`. The ruby interpreter doesn't seem to find these files. Please, kindly make them lowercase to fix this issue. :muscle:

Reply:
Thank you my friend, great find.